### DISCUSSION TITLE
constraint_XDISPLACEMENT

### DISCUSSION THREAD

**Discussion by EmreDumlu-UM** [Anura3D/Anura3D_OpenSource#153] [General]
Hi All,

Whenever I define two kinds of liquid via GID for Anura3D_2023, I get the same error  "can't read constraint_XDISPLACEMENT". However, I have never used that option so far. I can start the simulation by removing this input from .GOM file. Please see the attached figure.
![constraint_XDISPLACEMENT](https://github.com/Anura3D/Anura3D_OpenSource/assets/125078884/bbb07a21-cc30-451f-9d79-bd855e41fe3a)

**Comment by gdicarluccio** [comment]
Thanks for your feedback! The Problem Type has been just updated to fix this issue.
